dev-python/safety/safety-3.5.2.ebuild

```diff
@@ -0,0 +1,60 @@
+# Copyright 2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{11..13} )
+
+DISTUTILS_USE_PEP517="hatchling"
+
+inherit distutils-r1
+
+DESCRIPTION="Scan dependencies for known vulnerabilities and licenses."
+HOMEPAGE="https://safetycli.com"
+SRC_URI="https://github.com/pyupio/safety/archive/refs/tags/${PV}.tar.gz -> ${P}.gh.tar.gz"
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64"
+
+PATCHES=(
+	"${FILESDIR}/${PN}-${PV}-0001-feat-replace-nltk-with-pyspellchecker.patch"
+	"${FILESDIR}/${PN}-${PV}-0002-chore-add-tests-for-typosquatting.patch"
+	"${FILESDIR}/${PN}-${PV}-0003-fix-use-tomllib-for-newer-python-versions.patch"
+)
+
+RDEPEND="
+	>=dev-python/authlib-1.2.0[${PYTHON_USEDEP}]
+	>=dev-python/click-8.0.2[${PYTHON_USEDEP}]
+	>=dev-python/dparse-0.6.4[${PYTHON_USEDEP}]
+	>=dev-python/filelock-3.16.1[${PYTHON_USEDEP}]
+	>=dev-python/jinja2-3.1.0[${PYTHON_USEDEP}]
+	>=dev-python/marshmallow-3.15.0[${PYTHON_USEDEP}]
+	>=dev-python/packaging-21.0[${PYTHON_USEDEP}]
+	>=dev-python/psutil-6.1.0[${PYTHON_USEDEP}]
+	>=dev-python/pydantic-2.10.0[${PYTHON_USEDEP}]
+	<dev-python/pydantic-2.10.7[${PYTHON_USEDEP}]
```

Comment:
Where does this come from? It looks very arbitrary, and doesn't seem justified at all.